File: akka-samples/akka-sample-cluster-scala/src/main/scala/sample/cluster/transformation/TransformationApp.scala
package sample.cluster.transformation

object TransformationApp {

  def main(args: Array[String]): Unit = {
    // starting 2 frontend nodes and 3 backend nodes
    TransformationFrontend.main(Seq("2551").toArray)
    TransformationBackend.main(Seq("2552").toArray)
    TransformationBackend.main(Array.empty)
    TransformationBackend.main(Array.empty)
    TransformationFrontend.main(Array.empty)
  }

}

File: akka-samples/akka-sample-cluster-java/src/main/java/sample/cluster/transformation/TransformationApp.java
package sample.cluster.transformation;

public class TransformationApp {

  public static void main(String[] args) {
    // starting 2 frontend nodes and 3 backend nodes
    TransformationBackendMain.main(new String[] { "2551" });
    TransformationBackendMain.main(new String[] { "2552" });
    TransformationBackendMain.main(new String[0]);
    TransformationFrontendMain.main(new String[0]);
  }
}
